authorizations/authorization-common/src/main/java/org/apache/gravitino/authorization/common/PathBasedMetadataObject.java

```diff
@@ -92,15 +92,26 @@ public String toString() {
   private final String name;
   private final String parent;
   private final String path;
+  private final boolean isRecursive;
```

Comment:
I do like the name `isRecursive`, you may use `recursive` instead as there exists a method name `isRecursive()`

Reply:
Done.